packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -418,7 +418,30 @@ export class FilesService {
         return Number(result.rows[0].row_number) - 1;
     }
 
-    async createCopyOfFile(file: FileInterface, { inboxFolder }: { inboxFolder: FolderInterface }) {
+    async createCopyOfFile(
+        file: FileInterface,
+        {
+            inboxFolder,
+            targetFolder: inputTargetFolder,
+            targetScope: inputTargetScope,
+        }: {
+            /**
+             * @deprecated Use targetFolder instead
+             */
+            inboxFolder?: FolderInterface;
```

Comment:
I decided to deprecate the `inboxFolder` param and replace it with the `targetFolder` param.

The reason is:
In the DAM, inboxFolder refers to folder created while automatically copying files from one scope or environment to another while copying a page.

However, you can also copy a file into a non-inbox folder. So `targetFolder` is more fitting here IMO.